x-pack/plugins/apm/public/components/shared/transactions_table/index.tsx

```diff
@@ -141,6 +141,11 @@ export function TransactionsTable({
     type: ApmDocumentType.TransactionMetric,
   });
 
+  const shouldUseDurationSummary =
+    latencyAggregationType !== 'avg'
+      ? false
+      : preferred?.source?.isSummaryFieldAvailable ?? false;
```

Comment:
Do you need the ternary?

```suggestion
  const shouldUseDurationSummary = latencyAggregationType === "avg" && preferred?.source?.isSummaryFieldAvailable;
```